docs/example_ScalarField_Geotiff.html

```diff
@@ -11,19 +11,19 @@
 </head>
 
 <body>
-    <h1 class="title mapTitle">ScalarField GeoTIFF</h1>
```

Comment:
I think we should create a different example for this use case, something like **Multiple ScalarField from GeoTIFF** (adding it to /docs/index.html).

That way we don't loose different scale colors and popup description currently used in this example.

A perfect fit would be to add a (little) geotiff into `docs/data` with multiple bands for that new example (>= 2 bands, perhaps with 'time evolution' of the same variable?)

Reply:
That dataset should also be used also in a new spec, testing this method

Reply:
Example file: Multiple ScalarField from GeoTIFF  
Original example files reverted to previous versions.  

I'm using your `tz850.tiff` file, so I don't need to upload a new one.  
I put "Label <Number>" as layer names, and "Value" as popup text, because with the information existing in the tiff files I cannot obtain layer name. I don't know if this is even possible.

I let you to build the spec test file.